### PR TITLE
fix: style updates

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description

- Use **smaller** text size across tables
- Consistent margin for account action buttons

Other updates mentioned on PROD-2535 have been done in other PRs. While running on Safari, I didn't notice any style discrepancy, except the Account info table's rows border not going till the end.

### CODE REVIEW

File: src/users/account-actions/v2/PasswordHistory.jsx [hunk 1]
```diff
@@ -45,8 +45,9 @@ export default function PasswordHistory({
       {passwordStatus.passwordToggleHistory.length > 0 && (
       <Button
         id="toggle-password-history"
-        variant="outline-primary ml-1"
+        variant="outline-primary"
         onClick={() => openHistoryModal()}
+        className="ml-1 my-2"
```
Comment (Ali-D-Akbar) [inline src/users/account-actions/v2/PasswordHistory.jsx]: I think going for mr-1 mb-2 instead would be better for all the buttons. This way, it won't move the button from left and upwards.

File: src/users/account-actions/v2/ResetPassword.jsx [hunk 2]
```diff
@@ -18,8 +18,9 @@ export default function ResetPassword({
     <div>
       <Button
         id="reset-password"
-        variant="btn btn-danger ml-1"
+        variant="btn btn-danger"
         onClick={() => setResetPasswordModalIsOpen(true)}
+        className="ml-1 my-2"
```
Comment (Ali-D-Akbar) [inline src/users/account-actions/v2/ResetPassword.jsx]: Same for this as well.

File: src/users/account-actions/v2/TogglePasswordStatus.jsx [hunk 3]
```diff
@@ -29,6 +29,7 @@ export default function TogglePasswordStatus({
         id="toggle-password"
         variant={`${passwordStatus.status === PASSWORD_STATUS.USABLE ? 'danger' : 'primary'}`}
         onClick={() => setDisableUserModalIsOpen(true)}
+        className="ml-1 my-2"
```
Comment (Ali-D-Akbar) [inline src/users/account-actions/v2/TogglePasswordStatus.jsx]: Also this one as well.

**Review by Ali-D-Akbar — APPROVED** [review]
Just a suggestion on the margin stuff. The rest looks good.

**Review by azanbinzahid — APPROVED** [review]

**Pull request merged**